internal/oci/annotations.go

```diff
@@ -216,4 +216,17 @@ const (
 
 	// AnnotationSecurityPolicy is used to specify a security policy for opengcs to enforce
 	AnnotationSecurityPolicy = "io.microsoft.virtualmachine.lcow.securitypolicy"
+	// AnnotationProcessDumpLocation specifies a path inside of containers to save process dumps to. As
+	// the scratch space for a container is generally cleaned up after exit, this is best set to a volume mount of
+	// some kind (vhd, bind mount, fileshare mount etc.)
+	AnnotationProcessDumpLocation = "io.microsoft.processdumplocation"
```

Comment:
It's kind of confusing to me to have this annotation be usable on a pod configuration but correspond to the *containers'* dump location. 

Maybe we should rename it to "io.microsoft.container.processdumplocation"?

Reply:
It's a bit of a tricky spot as on linux it's basically a kernel setting that the containers will inherit, but yea that makes more sense to me.

Reply:
Renamed!